src/conf.py

```diff
@@ -125,6 +125,7 @@ def json_load(path):
 
 # NOTE: do not move to /tmp
```

Comment:
I added "cache_path" to start where you could specify a relative path such as ../  That can be adjusted if it should get its own absolute path format?